task5/src/main/java/com/example/demo111/GetName.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo111;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(name = "GetName", value = "/get")
+public class GetName extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String str = request.getQueryString();
+        if (str.length() <= 100) {
```

Comment:
What if the query string contains something different than the name?
Then your code won't work) 
Conventional code should be designed in such a way as to be "fail-proof". The amount of "unexpectedness"  must be reduced to a minimum. 
For the particular case `request.getAttribute("name")` would be a solution)